### PR TITLE
CDM-19: New QuoteElement and QuoteSlider components

### PR DESCRIPTION
**Pull request by catarinapm**
### QuoteElement

It can be **without** _author_ and **max lines** is 4.

The content is customizable (React JSX Element).

### QuoteSlider

[MOBILE]

https://github.com/cdm-mps/creativestudio/assets/129984310/3ff35731-bc31-42e3-91ba-c1206f1f6322

[WEB]

https://github.com/cdm-mps/creativestudio/assets/129984310/7c211247-fcae-43e1-a4d7-87f6ffc322fb

### CODE REVIEW

File: fe/app/components/Quote/QuoteElement.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import { QuoteElementProps } from "./Quote.models";
+
+const QuoteElement = ({
+  container,
+  text = {
+    content: <></>,
```
Comment (mariasbl) [inline fe/app/components/Quote/QuoteElement.tsx:6]: Remove the initial state, the content prop is required

**Review by mariasbl — CHANGES_REQUESTED** [review]

File: fe/app/components/Quote/Quote.models.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+export interface QuoteElementProps {
+  container?: { className: string };
+  text: {
+    content: React.JSX.Element;
+    author?: string;
+  };
```
Comment (danielaprado) [inline fe/app/components/Quote/Quote.models.tsx:6]: To simplify could you remove the nested props and update to something like this:
```
  className?: string;
  content: React.JSX.Element;
  author?: string;
```

File: fe/app/components/Quote/QuoteSlider.tsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import { QuoteSliderProps } from "./Quote.models";
+import QuoteElement from "./QuoteElement";
+
+const QuoteSlider = ({ elements }: QuoteSliderProps) => {
+  return (
+    <>
+      <div className="flex flex-no-wrap overflow-x-scroll test items-start mb-8 gap-x-[58px]">
+        {elements.map((e, i) => (
```
Comment (danielaprado) [inline fe/app/components/Quote/QuoteSlider.tsx:8]: For better context, could you rename the `elements` constant to `quotes` and the `e` to `quote`?

**Review by danielaprado — CHANGES_REQUESTED** [review]

File: fe/app/components/Quote/Quote.models.tsx (hunk 2 above)
Comment (catarinapm) [reply fe/app/components/Quote/Quote.models.tsx:6]: done

File: fe/app/components/Quote/QuoteElement.tsx (hunk 1 above)
Comment (catarinapm) [reply fe/app/components/Quote/QuoteElement.tsx:6]: you're right! thanks

File: fe/app/components/Quote/QuoteSlider.tsx (hunk 3 above)
Comment (catarinapm) [reply fe/app/components/Quote/QuoteSlider.tsx:8]: done!

**Review by danielaprado — APPROVED** [review]
LGTM 🚀

**Review by mariasbl — APPROVED** [review]

**Pull request merged**